README.md

```diff
@@ -10,33 +10,59 @@ Obelisk should be used as a dependency for external robot control code that is w
 3. Use Obelisk in a project that uses `pixi`.
 
 ### Initial Setup
```

Comment:
It might be worth clarifying that if the user wants to use Obelisk in a custom docker container (i.e. not the one we provide) then they will want to clone this **inside** the docker container and run the command that modify the "system level deps", because all of the conditional building is really for the dev container and presumably the user won't want to dev their robot stack in the Obelisk dev container. 

On the flip side maybe this is an overreach of the docs, but I think it might make the intended use clearer.

Reply:
Made a halfway attempt at commenting on this - if we need further clarity, let's resolve in a future PR